src/yamlWorker.ts

```diff
@@ -14,17 +15,17 @@ if (typeof fetch !== 'undefined') {
 }
 
 export interface YAMLWorker {
-  doValidation: (uri: string) => PromiseLike<ls.Diagnostic[]>;
+  doValidation: (uri: string) => Promisable<ls.Diagnostic[]>;
```

Comment:
What's the difference? I don't really understand.

Reply:
This is defined as

```ts
type Promisable<T> = T | PromiseLike<T>
```

This means that worker may return either promises or synchronous values, which is fine, as the monaco worker will turn the values into promises anyway.